doc/parent_child_spec.mld

```diff
@@ -175,17 +175,17 @@ installed and might be used by a different driver.
 In order for drivers to build consistent documentation for a package, the
 following convention should be followed.
 
-- [.mld] pages are installed in a package's [share] directory, under the
+- [.mld] pages and assets are installed in a package's [share] directory, under the
```

Comment:
Should assets be installed in a particular directory, or with a particular name?
With the proposed convention, assets with extension `.mld` are recognized as pages. But I think this is fine. If really needed, we could find some ways to work around the restriction with extension `.mld_`... ?